tests/Application/config/routes/sylius_admin.yaml

```diff
@@ -1,3 +1,6 @@
 sylius_admin:
     resource: "@SyliusAdminBundle/Resources/config/routing.yml"
     prefix: '/%sylius_admin.path_name%'
+
+setono_edit_order:
```

Comment:
Should be in its own file: `setono_sylius_order_edit.yaml`. Just to keep conventions